pkg/vdr/sov/vdr.go

```diff
@@ -0,0 +1,188 @@
+// +build ACAPyInterop
```

Comment:
this package is not sov - this would be confusing.

Reply:
Addressed this by removing the "fake sov" interop vdr and refactoring the conversion logic into didexchange itself.